code/modules/organs/limbs.dm

```diff
@@ -558,6 +558,17 @@ Note that amputating the affected organ does in fact remove the infection from t
 	if((limb_status & LIMB_ROBOT)) //Robotic limbs don't heal or get worse.
 		return
 
+	if(limb_status & LIMB_SPLINTED) //Splints gradually break over time.
+		if(splint_time <= 0)
+			remove_limb_flags(LIMB_SPLINTED)
+			to_chat(owner, span_userdanger("The splint on your [display_name] comes apart!"))
+		else
+			if(world.time - last_move_time > 30)
```

Comment:
you didnt test this did you?

Reply:
I literally just made this. It still needs testing. I'm like 80% sure it doesn't compile.

Reply:
fixed